scripts/openwrt_upload.sh

```diff
@@ -6,6 +6,6 @@ export ROUTER_IP=192.168.10.1
 bash scripts/openwrt_build_$TARGET.sh --features rita_bin/development
 set +e
 ssh root@$ROUTER_IP killall -9 rita
-set -e
+set -e 
```

Comment:
extra change, remove the trailing space and amend to drop this file from the commit.

Reply:
fixed